### PR TITLE
Feature/docs analytics reorg

### PR DESCRIPTION
**Pull request by lenapivotal**
Please review staging copy of new analytics menu, after General Information:
http://docs-lena-analytics-reorg.cfapps.io/7-0/main/index.html

For previous review comments (from David and Lisa), please see older PR 
https://github.com/greenplum-db/gpdb/pull/9250

### CODE REVIEW

File: gpdb-doc/dita/analytics/overview.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="gp_integrated_analytics">
+  <title>Overview of Greenplum Database Integrated Analytics</title>
+  <body>
+        <p>Greenplum offers a unique combination of a powerful massively parallel processing (MPP)
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Add comma after: powerful,

File: gpdb-doc/dita/analytics/overview.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      database and advanced data analytics. This combination creates an ideal framework for data
+      scientists, data architects and business decision makers to explore artificial intelligence
+      (AI), machine learning, deep learning, text analytics, and geospatial.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: geospatial -> geospatial analytics

File: gpdb-doc/dita/analytics/overview.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    
+    <p>This topic covers:</p>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Since this topic is so short (as an overview should be), maybe we can omit this intro and just continue with the topics?

File: gpdb-doc/dita/analytics/overview.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+      <p><xref href="http://gptext.docs.pivotal.io/330/topics/intro.html" format="dita">Pivotal
+        GPText</xref> enables processing mass quantities of raw text data 
+        (such as social media feeds or e-mail databases) into mission-critical information that 
+        guides project and business decisions. GPText joins the Greenplum Database massively parallel-processing 
+        database server with Apache SolrCloud enterprise search. GPText includes powerful text search as well as
+        support for text analysis.</p>
+    </section>
+    
+    <section id="pr_lang_and_ext">
+        <title>Programming Language Extensions</title>The Greenplum Database distribution supports
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Seems like this intro sentence should better tie things into analytics?  Something like "Greenplum database supports a variety of procedural languages that you can use for programming database analytics.

File: gpdb-doc/dita/analytics/overview.xml [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        the following procedural languages. Refer to the linked documentation for installation and
+        usage instructions.<p><xref href="pl_container.xml">PL/Container</xref>, <xref
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Make this into a bullet list.

File: gpdb-doc/dita/analytics/overview.xml [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+            >PL/Perl</xref>, <xref href="pl_sql.xml" format="dita">PL/pgSQL</xref>, <xref
+            href="pl_python.xml" format="dita">PL/Python</xref>, <xref href="pl_r.xml" format="dita"
+            >PL/R</xref>.</p>
+      </section>
+   </body>
+    </topic>
+    
+    
+    <topic id="why_gp_analytics">
+      <title>Why Greenplum Database in Integrated Analytics</title>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Maybe reword to "Why Use Greenplum Integrated Analytics?"

File: gpdb-doc/dita/analytics/overview.xml [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+      <body>
+      <p>The importance of advanced analytics in its various forms is growing rapidly in enterprise
+        computing. Key enterprise data typically resides in relational and document form and it is
+        inefficient to copy data between systems to perform analytical operations. Greenplum is able
+        to run both traditional and advanced analytics workloads in-database. This integrated
+        capability greatly reduces the cost and the silos created by procuring and maintaining
+        multiple tools and libraries.</p>
+        <p>Greenplum Database advanced analytics can be used to address a wide variety of problems
+        in many verticals including automotive, finance, manufacturing, energy, government,
+        education, telecommunications, on-line and traditional retail.</p>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: , on-line -> , and on-line

File: gpdb-doc/dita/analytics/overview.xml [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+        <p>The Greenplum analytics capabilities allow you to:</p>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: The Greenplum -> Greenplum

File: gpdb-doc/dita/analytics/overview.xml [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+    <ul id="ul_o3q_5n4_yjb">
+      <li>Analyze a multitude of data types – structured, text, geospatial, and graph – in a single environment, 
+        which can scale to petabytes and run algorithms designed for parallelism.  </li>
+      <li>Leverage existing SQL knowledge: Pivotal Greenplum can run dozens of statistical,
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: I think "Pivotal" should be removed in this sentence, since the capabilities would be available in open source Greenplum also.

File: gpdb-doc/dita/analytics/overview.xml [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+        <p>The Greenplum analytics capabilities allow you to:</p>
+    <ul id="ul_o3q_5n4_yjb">
+      <li>Analyze a multitude of data types – structured, text, geospatial, and graph – in a single environment, 
+        which can scale to petabytes and run algorithms designed for parallelism.  </li>
+      <li>Leverage existing SQL knowledge: Pivotal Greenplum can run dozens of statistical, 
+        machine learning, and graph methods, via SQL.</li>
+      <li>Train more models in less time by taking advantage of the parallelism in the MPP
+        architecture and in-database analytics.</li>
+      <li>Access the data where it lives, therefore integrate data and analytics in one place. 
+        Pivotal Greenplum is infrastructure-agnostic and runs on  bare metal, private cloud, and public
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: Remove "Pivotal" here.

File: gpdb-doc/dita/analytics/overview.xml [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+      <li>Analyze a multitude of data types – structured, text, geospatial, and graph – in a single environment, 
+        which can scale to petabytes and run algorithms designed for parallelism.  </li>
+      <li>Leverage existing SQL knowledge: Pivotal Greenplum can run dozens of statistical, 
+        machine learning, and graph methods, via SQL.</li>
+      <li>Train more models in less time by taking advantage of the parallelism in the MPP
+        architecture and in-database analytics.</li>
+      <li>Access the data where it lives, therefore integrate data and analytics in one place. 
+        Pivotal Greenplum is infrastructure-agnostic and runs on  bare metal, private cloud, and public 
+        cloud deployments.</li>
+      <li>Use a multitude of data extensions. Greenplum supports Apache Kafka connector, extensions
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: supports Apache Kafka connector -> provide an Apache Kafka integration.  I think the rest of this bullet needs a bit of work, since PXF is the extension that provides hdfs/hive/base.  Also, only the Kafka part of this should be conditionalized for Pivotal-only builds.  Rough suggestion:

Use a multitude of data extensions for reading/writing data from/to a variety of data source. Greenplum supports an Apache Kafka integration, and Pivotal Extension Framework (PXF) for working with HDFS, Hive, HBase, and Amazon S3 data objects.

File: gpdb-doc/dita/analytics/overview.xml [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+            href="https://gpdb.docs.pivotal.io/6-1/pxf/intro_pxf.html" format="html"
+            scope="external">Platform Extension Framework (PXF)</xref>, which provides
+            <i>connectors</i> that enable you to access data stored in sources external to your
+          Greenplum Database deployment.</li>
+      <li>Use familiar and leading BI and advanced analytics software that are ODBC/JDBC compatible, or have native integrations,
+        including SAS, IBM Cognos, SAP Analytics Solutions, Qlik, Tableau, Apache Zeppelin, and Jupyter.</li>
+      <li>Execute deep learning algorithms using popular frameworks like Keras and TensorFlow in an
+          MPP relational database, with GPU (Graphical Processing Unit) acceleration.</li>
+      <li>Use containers capable of isolating executors from the host OS. Greenplum PL/Container implements
+        a trusted language execution engine which permits customized data science workloads or environments created
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: which -> that

File: gpdb-doc/dita/analytics/text.xml [hunk 13]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="topic1" xml:lang="en">
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/text.xml]: This topic will need a bit of work to function in both open source and pivotal environments, since GPText info should be conditionalized for Pivotal greenplum.  It would be fine to take the simplest approach and just conditionalize the one bullet.

File: gpdb-doc/dita/analytics/text.xml [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+  <title id="ij138244">Text Analytics and Search</title>
+  <body>
+    <p>Greenplum Database offers two different methods for text search and analytics, <b>Greenplum
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/text.xml]: two -> these

File: gpdb-doc/dita/analytics/text.xml [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+        Database full text search</b> and <b>Pivotal GPText</b>.</p>
+    <section>
+      <title>Greenplum Database Full Text Search</title>
+    </section>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/text.xml]: Move this closing /section marker to the end of the text (after the following paragraph).

File: gpdb-doc/dita/analytics/text.xml [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+    <p>Greenplum Database text search is PostgreSQL text search ported to the Greenplum Database MPP
+      platform.  Greenplum Database text search is immediately available to you, with no need to
+      install and maintain additional software. For full details on this topic, see <xref
+        href="../admin_guide/textsearch/intro.xml" format="dita" scope="peer">Greenplum Database
+        text search</xref>.</p>
+    <section>
+      <title>Pivotal GPText</title>
+    </section>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/text.xml]: Move this closing /section marker as well, and then conditionalize the entire section as otherprops=pivotal

File: gpdb-doc/dita/gpdb-webhelp.ditamap [hunk 17]
```diff
@@ -2,6 +2,7 @@
 <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
 <map><title>Greenplum Database</title>
 	<topicref href="common/gpdb-features.xml" format="dita"/>
```
Comment (dyozie) [inline gpdb-doc/dita/gpdb-webhelp.ditamap]: This .ditamap file also needs a top-level link to return to the main Greenplum 7.0 documentation.  You can copy this from any other .ditamap, like the installation .ditamap.  There should be 2 entries, conditionalized for open source and pivotal builds.  Let me know if you need help with this.

File: gpdb-doc/dita/analytics/overview.xml [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+        Pivotal Greenplum is infrastructure-agnostic and runs on  bare metal, private cloud, and public 
+        cloud deployments.</li>
+      <li>Use a multitude of data extensions. Greenplum supports Apache Kafka connector, extensions
+          for HDFS, Hive, and HBase as well as reading/writing data from/to cloud storage, including
+          Amazon S3 objects. Review the capabilities of Greenplum <xref
+            href="https://gpdb.docs.pivotal.io/6-1/pxf/intro_pxf.html" format="html"
+            scope="external">Platform Extension Framework (PXF)</xref>, which provides
+            <i>connectors</i> that enable you to access data stored in sources external to your
+          Greenplum Database deployment.</li>
+      <li>Use familiar and leading BI and advanced analytics software that are ODBC/JDBC compatible, or have native integrations,
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/overview.xml]: This list item should be conditionalized for Pivotal, since it covers a lot of proprietary integrations.

File: gpdb-doc/dita/analytics/overview.xml (hunk 1 above)
Comment (lenapivotal) [reply gpdb-doc/dita/analytics/overview.xml]: agreed

File: gpdb-doc/dita/analytics/overview.xml (hunk 3 above)
Comment (lenapivotal) [reply gpdb-doc/dita/analytics/overview.xml]: Removed the list

File: gpdb-doc/dita/analytics/overview.xml (hunk 5 above)
Comment (lenapivotal) [reply gpdb-doc/dita/analytics/overview.xml]: Created bullet list

File: gpdb-doc/dita/gpdb-webhelp.ditamap (hunk 17 above)
Comment (lenapivotal) [reply gpdb-doc/dita/gpdb-webhelp.ditamap]: I have made a similar edit to the install guide ditamap. But we might need to revisit this file.
